.github/workflows/black.yml

```diff
@@ -0,0 +1,14 @@
+name: Black
+
+on: push
+
+jobs:
+  black:
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+      - uses: psf/black@stable
+        with:
+          options: "--check"
```

Comment:
This version of black will eventually get out of step with the Poetry-installed version, and silently cause problems. That's why the check was in the `python-ci` action -- so the same version is used by both CI and local developers.